generator/util/query.py

```diff
@@ -276,14 +276,13 @@ def get_inline_parents(class_object):
 
 def get_parent_class(class_object):
     """
-    NB Open Github Issue: Revise query get_parent_class code #36
-
     Return the parent class of this class.
-    if it is not given we assume it is a child of a plugin
-    and get the base of the plugin.
+    If a class does not explicitly declare its parent then it's parent may
+    be the outside this package i.e. a plugin object. In this case it will be
```

Comment:
typo, unnecessary 'the'

Reply:
???

Reply:
'be the outside this package' - no need for 'the' here